forge-gui/res/cardsfolder/upcoming/rose_tyler.txt

```diff
@@ -0,0 +1,13 @@
+Name:Rose Tyler
+ManaCost:1 W
+Types:Legendary Creature Human
+PT:2/2
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ X | AddToughness$ X | Description$ CARDNAME gets +1/+1 for each time counter on it.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Bad Wolf — Whenever CARDNAME attacks, put a time counter on it for each suspended card you own and each other permanent you control with a time counter on it.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ TIME | CounterNum$ Y
+SVar:X:Count$CardCounters.TIME
+SVar:Y:Count$Valid Card.suspended+YouOwn,Permanent.Other+YouCtrl+counters_GE1_TIME
```

Comment:
Isn't working properly for the suspended cards.

Reply:
default is battlefield only
check e.g. _Crackling Drake_